lighthouse-cli/printer.ts

```diff
@@ -17,45 +17,31 @@
 
 'use strict';
 
-const fs = require('fs');
-const ReportGenerator = require('../lighthouse-core/report/report-generator');
-const Formatter = require('../lighthouse-core/formatters/formatter');
-const log = require('../lighthouse-core/lib/log');
-
 /**
  * An enumeration of acceptable output modes:
- * <ul>
- *   <li>'pretty': Pretty print the results</li>
- *   <li>'json': JSON formatted results</li>
- *   <li>'html': An HTML report</li>
- * </ul>
- * @enum {string}
+ *   'pretty': Pretty print the results
+ *   'json': JSON formatted results
+ *   'html': An HTML report
  */
-const OUTPUT_MODE = {
-  pretty: 'pretty',
-  json: 'json',
-  html: 'html'
+enum OutputMode { pretty, json, html };
```

Comment:
@paulirish this removes the layer of indirection :) .. using the nice reverse lookup enum name trick @mprobst tipped me off to :)